pom.xml

```diff
@@ -25,13 +25,13 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
-            <version>2.0.9</version>
+            <version>2.0.16</version>
         </dependency>
 
         <dependency>
             <groupId>com.ibm.eventstreams.connect</groupId>
             <artifactId>kafka-connect-mq-source</artifactId>
-            <version>1.3.4</version>
+            <version>2.1.0</version>
```

Comment:
I think latest source version is 2.0.1

Reply:
That's true for the internal release, but I did the build against the public external one at https://github.com/ibm-messaging/kafka-connect-mq-source/releases/tag/v2.1.0

Reply:
My bad I thought the latest version of mq source in OSS was 2.0.1.

The checks are failing because they can't pull the latest MQ Source Connector JAR from maven. If you don't mind, could you please review this PR(I assume that is the right way to do it)? It adds instructions on how to pull and add the latest MQ Source JARs, and it also updates the GitHub Action to pull in the latest JAR before running the checks.

Please note that the Maven target is set to JDK 17, so I have updated the builds to match that. If this is not intended, I can raise a PR that would change the build to use JDK 8 instead.

https://github.com/ibm-messaging/kafka-connect-xml-converter/pull/5